ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_searching.py

```diff
@@ -383,3 +395,32 @@ def test_numpy_nanargmin(
         axis=axis,
         keepdims=keep_dims,
     )
+
+    
+# extract
+@handle_frontend_test(
+    fn_tree="numpy.extract",
+    broadcastables=_broadcastable_duo(),
+)
+def test_numpy_extract(
+    broadcastables,
+    as_variable,
+    num_positional_args,
+    native_array,
+    frontend,
+    fn_tree,
+    on_device,
+):
+    a, cond, dtype = broadcastables
+    helpers.test_frontend_function(
```

Comment:
Currently the test is failing with the error:
```
=================================== FAILURES ===================================
______ test_numpy_extract[cpu-ivy.functional.backends.torch-False-False] _______
ivy/exceptions.py:118: in new_fn
    return fn(*args, **kwargs)
ivy/functional/backends/torch/data_type.py:149: in as_ivy_dtype
    return ivy.Dtype(ivy_dtype_dict[dtype_in])
E   TypeError: unhashable type: 'list'
```

See the test output of the first check running on this PR for the full stack trace (`run_tests_1`) for more details and the decorator to add to the test locally to reproduce the failure if you are currently not seeing it.